docs/usage.rst

```diff
@@ -8,6 +8,14 @@ First, make sure you're using Python 3,
 you've :doc:`installed the bigchaindb_driver Python package <quickstart>`,
 and you've :doc:`connected to a BigchainDB node or cluster <connect>`.
 
+For the sake of these examples, we'll assume:
+
+.. ipython::
+
+    In [0]: from bigchaindb_driver import BigchainDB
+
+    In [0]: bdb = BigchainDB('http://bdb-server:9984/api/v1')
```

Comment:
The whole point of the new `connect.rst` page was to allow for all the diverse possible URL strings that could go inside the first argument of `BigchainDB()`. Why do we need to assume that it's `'http://bdb-server:9984/api/v1'`? I already told them to assume that they're connected to a BigchainDB node or cluster, with a link to the `connect.rst` page.

Maybe the solution is to edit the `connect.rst` page to tell them to use IPython and to put IPython code blocks everywhere on that page?